packages/vite-extensions/vite-website-cache-paths.ts

```diff
@@ -0,0 +1,83 @@
+import { readdirSync, statSync, writeFileSync } from 'node:fs';
+import { join } from 'node:path';
+
+const patternsToNotCache = [
+	'/package.json',
+	'/README.md',
+	'/index.cjs',
+	'/index.d.ts',
+	'/builder/builder.html',
+	'/gutenberg.css',
+	'/gutenberg.html',
+	'/.htaccess',
+	'/ogimage.png',
+	'/previewer.css',
+	'/wordpress-importer.zip',
+	'/wordpress.html',
+	'/wordpress.svg',
+	'/.DS_Store',
+	'/.htaccess',
+	/^\/wp-[\w+((\.\d+)?)]+\/.*/, // WordPress static asset folders
+	/^\/demos\/.*/, // Demos
+	/\/.*\.php$/, // PHP files used by the website server
```

Comment:
Let's file an issue to explore caching what we get from the plugin proxy – maybe it's a terrible idea, but maybe it would make some Blueprints work offline

Reply:
Moving conversation here https://github.com/WordPress/wordpress-playground/pull/1600/files#r1675670136